src/schema/friend/list.ts

```diff
@@ -0,0 +1,34 @@
+import { Type } from "@sinclair/typebox";
+
+import { defineEndpoint } from "@/generator-helpers.js";
+import { unixTime } from "@/schema/definitions/unixTime";
+import { userId } from "@/schema/definitions/userId";
+
+export default defineEndpoint({
+    source: "user",
+    target: "server",
+    description: "Retrieve the status of your friendlist",
+    request: {
+        data: Type.Object({
+            friends: Type.Array(
+                Type.Object({
+                    userId: Type.Ref(userId),
+                    addedAt: Type.Ref(unixTime),
+                })
+            ),
+            outgoing_pending_requests: Type.Array(
```

Comment:
All property names should be camel case. We should figure out a linting rule for this....